package.json

```diff
@@ -4,7 +4,8 @@
   "description": "Various tools for using and integrating with Swagger.",
   "main": "index.js",
   "scripts": {
-    "test": "gulp"
+    "test": "gulp",
+    "test-debug": "node --debug-brk node_modules/.bin/gulp"
```

Comment:
I would rather not to change the default test-command to explicit call the gulp script. npm should take care of script path resolution.
the reason i do manually for *test-debug* is because gulp has no option to enable node's debug mode. this is more of a workaround than a reliable solution i think.